### PR TITLE
CU-2wv5xwu_Tech_Manual

### PR DESCRIPTION
**Pull request by Neoramses1984**
tech manual

### CODE REVIEW

**Comment by Neoramses1984** [comment]
Task linked: [CU-2wv5xwu Realizar manual técnico de la calculadora fault location](https://app.clickup.com/t/2wv5xwu)

**Review by cmacana — CHANGES_REQUESTED** [review]

File: documentation/manual_tecnico.md [hunk 1]
```diff
@@ -1,121 +1,135 @@
-# MANUAL TÉCNICO 
-## LOCALIZADOR DE FALLAS
-![logo_prueba](logo_enphoria.png "Logo Enphoria")
```
Comment (Neoramses1984) [inline documentation/manual_tecnico.md:3]: the image was changed for a new front page with all logos.

File: documentation/manual_tecnico.md [hunk 2]
```diff
@@ -1,121 +1,135 @@
-# MANUAL TÉCNICO 
-## LOCALIZADOR DE FALLAS
-![logo_prueba](logo_enphoria.png "Logo Enphoria") 
-![logo_prueba](logo_zepben.png "Logo Zepben")
-![logo_prueba](logogitud.png "Logo GITUD")
-### METODOLOGÍA PROPUESTA
-El método de localización de fallas que proponemos combina la información de la red del feeder junto con las mediciones de corriente para la posible localización de fallas. En primer lugar, el método utiliza la magnitud de las corrientes adquiridas durante la condición de fallo por el reconectador o el disyuntor para determinar el tipo de fallo y las fases involucradas. Se realiza un rastreo aguas abajo del reconectador y se calcula la impedancia equivalente total de cada línea aguas abajo del alimentador. La corriente de fallo se calcula para todas líneas aguas abajo y se compara con la corriente de fallo registrada para obtener la ubicación de la falla basada en la coincidencia de la corriente de falla.
```
[the remark this answers is not in the document]
Comment (Neoramses1984) [reply documentation/manual_tecnico.md:7]: done

File: documentation/manual_tecnico.md [hunk 3]
```diff
@@ -1,121 +1,135 @@
...
-
-Los principales pasos del algoritmo se describen a continuación siguiente y se muestran en la Figura 1.
-
-![logo_prueba](imagen1.png "Diagrama de flujo del algoritmo propuesto") Fig. 1: Diagrama de flujo del algoritmo propuesto.
-
-A. Procesador de fallas de entrada
-
-Un reconectador o un disyuntor aguas arriba de la falla se dispara cuando se produce una falla en una línea. El reconectador o el disyuntor disparado registra la magnitud RMS de la corriente de falla en cada fase (IFa, IFb, IFc) y la tierra (IFG) e informa
```
[the remark this answers is not in the document]
Comment (Neoramses1984) [reply documentation/manual_tecnico.md:15]: done

File: documentation/manual_tecnico.md [hunk 4]
```diff
@@ -1,121 +1,135 @@
...
-al centro de control. En función de estas entradas, se desarrolla un algoritmo.
-
-B. Trazado de la trayectoria aguas abajo
-
-Se realiza un rastreo aguas abajo del reconectador disparado o del disyuntor. El rastreo se detiene cuando se descubre un reconectador, un fusible o un disyuntor, o cuando se alcanza el final de la línea. La Fig. 2 ilustra el trazado descendente de las líneas coloreadas en rojo. El terminal de cabecera es el lugar de supervisión, es decir, el reconectador o el disyuntor disparado, y el trazado comienza desde aquí. El trazado descendente se detiene cuando se alcanza un reconectador (después del nodo 3), un fusible (después del nodo 2) o el final de la línea.
```
[the remark this answers is not in the document]
Comment (Neoramses1984) [reply documentation/manual_tecnico.md:20]: done

File: documentation/manual_tecnico.md [hunk 5]
```diff
@@ -1,121 +1,135 @@
...
...enan en la base de datos de la empresa de servicios públicos. Algunos pocos feedeers de la red de distribución de NSW se cargan en el servidor de energy workbench (EWB). El servidor EWB es el servidor principal que proporciona la interfaz de programación de aplicaciones (API) para acceder a la red y los datos de series temporales, así como a las aplicaciones web que permiten visualizaciones y otras capacidades de alto nivel conectadas a los datos de la API. Los resultados se visualizan en GIS utilizando el cliente del servidor de aplicaciones de evolución (EAS). El algoritmo propuesto se prueba en veintiún eventos de fallas del mundo real recopilados de una de las empresas de servicios públicos de distribución en Australia para evaluar la efectividad. Cada evento cuenta con datos de seguimiento como la magnitud de la corriente en cada fase y la tierra. El equipo de servicios públicos ha patrullado el área y ha descubierto las ubicaciones reales de las fallas. Las ubicaciones reales de las fallas se comparan con las ubicaciones identificadas determinadas por el algoritmo propuesto. Cuatro de los eventos se han explicado en esta sección seguido de un resumen de los eventos restantes.
```
[the remark this answers is not in the document]
Comment (Neoramses1984) [reply documentation/manual_tecnico.md:69]: done

[the remark this answers is not in the document]
Comment (Neoramses1984) [reply documentation/manual_tecnico.md:69]: done

File: documentation/manual_tecnico.md [hunk 6]
```diff
@@ -100,7 +100,9 @@ Fig. 4: Distribución de la región de fallo según la expectativa de fallo
 
 Se observa que alrededor del 50 por ciento de la falla se localiza con una expectativa de falla de 95-100 por ciento con el método propuesto. Aproximadamente el 12,5 por ciento de la falla se encuentra con una expectativa de falla del 90-95 por ciento, seguido por el 20,83 por ciento con una expectativa de falla del 75-90 por ciento. El método propuesto no es capaz de localizar la falla en alrededor del 16,67 por ciento de la falla como las corrientes en las fases sin falla y de la carga que son ignoradas. Tampoco se considera la incertidumbre o inexactitud de los datos en el método propuesto.
```
[the remark this answers is not in the document]
Comment (Neoramses1984) [reply documentation/manual_tecnico.md]: done

[the remark this answers is not in the document]
Comment (Neoramses1984) [reply documentation/manual_tecnico.md]: done

File: documentation/manual_tecnico.md [hunk 7]
```diff
@@ -1,121 +1,135 @@
...
...enan en la base de datos de la empresa de servicios públicos. Algunos pocos feedeers de la red de distribución de NSW se cargan en el servidor de energy workbench (EWB). El servidor EWB es el servidor principal que proporciona la interfaz de programación de aplicaciones (API) para acceder a la red y los datos de series temporales, así como a las aplicaciones web que permiten visualizaciones y otras capacidades de alto nivel conectadas a los datos de la API. Los resultados se visualizan en GIS utilizando el cliente del servidor de aplicaciones de evolución (EAS). El algoritmo propuesto se prueba en veintiún eventos de fallas del mundo real recopilados de una de las empresas de servicios públicos de distribución en Australia para evaluar la efectividad. Cada evento cuenta con datos de seguimiento como la magnitud de la corriente en cada fase y la tierra. El equipo de servicios públicos ha patrullado el área y ha descubierto las ubicaciones reales de las fallas. Las ubicaciones reales de las fallas se comparan con las ubicaciones identificadas determinadas por el algoritmo propuesto. Cuatro de los eventos se han explicado en esta sección seguido de un resumen de los eventos restantes.
```
[the remark this answers is not in the document]
Comment (Neoramses1984) [reply documentation/manual_tecnico.md]: done

**Review by Neoramses1984 — COMMENTED** [review]
changes done

**Comment by Neoramses1984** [comment]
tech manual with corrections

**Pull request closed without merging**